parallel/parallel_collections.py

```diff
@@ -62,7 +68,7 @@ def foreach(self, func):
         
     def filter(self, pred):
         _filter = _Filter(pred)
-        return self.__class__((i for i in _map(_filter, self, ) if i is not None))
+        return self.__class__((i[1] for i in _map(_filter, self, ) if i[0]))
```

Comment:
Did you consider using [named tuples](https://docs.python.org/2/library/collections.html#collections.namedtuple) instead of the index access, for better readability?